### PR TITLE
Workflow label

### PR DESCRIPTION
**Pull request by sean-m-sullivan**
##### SUMMARY
Add Label option to workflow_job_template module

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - Collection

##### AWX VERSION
awx: 14.1.0

```

### CODE REVIEW

**Comment by sean-m-sullivan** [comment]
Did more testing, since labels are organization based, added organizations to the search parameters on both workflows and templates, This assumes that the user has put in the organization for templates, which is based off of project. 

Will revisit in morning, leaving this code commented out, until #7567 is resolved.

File: awx_collection/plugins/modules/tower_job_template.py [hunk 1]
```diff
@@ -481,6 +481,12 @@ def main():
         association_fields['labels'] = []
         for item in labels:
             association_fields['labels'].append(module.resolve_name_to_id('labels', item))
+# Code to use once Issue #7567 is resolved
+#            search_fields = {'name': item}
+#            if organization:
+#                search_fields['organization'] = organization_id
```
Comment (AlanCoding) [inline awx_collection/plugins/modules/tower_job_template.py]: trailing whitespace is causing some errors here

File: awx_collection/plugins/modules/tower_workflow_job_template.py [hunk 2]
```diff
@@ -247,6 +253,18 @@ def main():
...
 
+    labels = module.params.get('labels')
+    if labels is not None:
+        association_fields['labels'] = []
+        for item in labels:
+            association_fields['labels'].append(module.resolve_name_to_id('labels', item))
+# Code to use once Issue #7567 is resolved
+#            search_fields = {'name': item}
+#            if organization:
+#                search_fields['organization'] = organization_id
```
Comment (AlanCoding) [inline awx_collection/plugins/modules/tower_workflow_job_template.py]: more trailing whitespace here

**Comment by AlanCoding** [comment]
It would be good to rebase this, but you should wait for the get_one PR to merge

**Comment by AlanCoding** [comment]
Running checks one more time, 🤞  that passing, I'd love to get this in.

**Comment by sean-m-sullivan** [comment]
believe the check has passed.

**Review by AlanCoding — APPROVED** [review]

**Comment by AlanCoding** [comment]
regate

**Comment by sean-m-sullivan** [comment]
regate

**Comment by AlanCoding** [comment]
regate

**Pull request merged**